README.md

```diff
@@ -13,12 +13,22 @@ docker run -p 80:80 -e PORT=80 -e JWT_SECRET=test uoftboard
 
 Setup needs to be done in order to use the type definitions from the server in our client files.
 
-Make sure that "declaration": true is set in server tsconfig and perform yarn build in server
+Make sure that `"declaration": true` is set in server tsconfig and perform `yarn build` in server dir
```

Comment:
```suggestion
Before continuing, make sure you `git pull` and then run `yarn build` in server dir
```
``"declaration": true`` is already set in the codebase. This is a bit redundant.

Reply:
agreed